src/SwedbankPay.Sdk/Common/Amount.cs

```diff
@@ -0,0 +1,113 @@
+﻿using System;
+using System.Globalization;
+
+namespace SwedbankPay.Sdk.Common
+{
+    public class Amount : IEquatable<Amount>, IComparable<Amount>, IComparable
```

Comment:
Codacy found an issue: [When implementing IComparable<T> or IComparable, you should also override <, >, <=, >=, ==, !=.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=6290774)